assets/js/base/context/container-query-context.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { createContext, useContext } from '@wordpress/element';
+import { useContainerQueries } from '@woocommerce/base-hooks';
+import classNames from 'classnames';
+
+const ContainerQueryContext = createContext( {
+	containerQueryClassName: '',
+	isMobile: false,
+	isSmall: false,
+	isMedium: false,
+	isLarge: false,
```

Comment:
If all of these are false by default, how will a consumer that implements `useContainerQueryContext` outside of the context tree behave? I think at least one of these values should be true by default.

Reply:
Before the first render we don't know the container size, so I think it makes sense that all of them are `false` by default. Consumers can check `isLoading` value to know if the component was rendered or not. Actually that's how it's done in this PR in `sidebar/order-summary.js`.

Reply:
> Actually that's how it's done in this PR in sidebar/order-summary.js.
Ahh right!